content/en/blog/_posts/2021-0420-cncf-dan.md

```diff
@@ -0,0 +1,196 @@
++++ 
+title = "Thank You Dan Kohn" 
+date = 2021-03-19 
+author = ["Riaan Kleinhans"]
```

Comment:
CI tests fail.

https://app.netlify.com/sites/kubernetes-io-master-staging/deploys/608062c21a540a00089913b4

```
2:38:14 AM: Error: Error building site: failed to render pages: render of "page" failed: execute of template failed: template: blog/single.html:2:3: executing "main" at <.Render>: error calling Render: "/opt/build/repo/content/en/blog/_posts/2021-0420-cncf-dan.md:5:62": failed to execute template ["content"] v: "/opt/build/repo/themes/docsy/layouts/blog/content.html:5:62": execute of template failed: template: blog/content.html:5:62: executing "blog/content.html" at <markdownify>: error calling markdownify: unable to cast []string{"Riaan Kleinhans"} of type []string to string
```

```suggestion
author = "Riaan Kleinhans"
```
This might solve the problem.

Reply:
I think that the authorship line usually goes just outside the frontmatter, something like this:

```
**Author**: Riaan Kleinhans
```

Reply:
Docsy's blog template doesn't seem to support multiple authors with attributions, so we write them inline.